frontend/src/hooks/use-search/use-search.hook.ts

```diff
@@ -0,0 +1,34 @@
+import { useAppDispatch, useSearchParams } from 'hooks/hooks';
```

Comment:
it isn't a reusable hook. move it to place where you use it

Reply:
I've used it for search and for filtering by Category. Not sure where to move it in this case ... Components folder?

Reply:
as we can have `helpers` folder inside component's folder, we can have `hooks` folder as well

Reply:
as we can have `helpers` folder inside component's folder, we can have `hooks` folder as well

Reply:
@what1s1ove Sorry, but I do not really get it.  Hook is used in 2 components: Header and  Category-list. Creating another 'hooks' folder in Components doesn't seem right, as well as creating 'hooks' folder for each separate component. Can you please explain what needs to be done here?

Reply:
oh sorry. got it now